bin/watch.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+const gri = require('gaze-run-interrupt');
+
+const binList = [
+  // 'bin/analyze',
+  // 'bin/preflight',
+  'bin/support-bundle',
+  // 'bin/collect'
+]
+const makeList = [
+  // 'analyze',
+  // 'preflight',
+  'support-bundle',
+  // 'collect'
+]
+
+const commands = [
+  // {
+  //   command: 'rm',
+  //   args: binList,
+  // },
+  {
+    command: 'make',
```

Comment:
You can try `make build`. It should take the same amount of time. It builds all binaries in parallel.